travis/.rubocop_rspec_base.yml

```diff
@@ -114,10 +114,61 @@ SpaceAroundEqualsInParameterDefault:
 StringLiterals:
   Enabled: false
 
+# By convention, RSpec examples use expect { ... } rather than expect do ... end
+Style/Blocks:
+  Exclude:
+    - spec/**/*
+
+# In tests, it is convenient to define a test class like RSpec::Something on a single line
+Style/ClassAndModuleChildren:
+  Exclude:
+    - spec/**/*
+
+# Our tests use compact class definitions to save "real estate" on the scren.
+Style/EmptyLineBetweenDefs:
+  Exclude:
+    - spec/**/*
+Style/EmptyLinesAroundAccessModifier:
+  Exclude:
+    - spec/**/*
+
+Style/GlobalVars:
+  Exclude:
+    - spec/**/*
+    - benchmarks/**/*
+
+# Allow use of: expect { ... }.to raise_error(e) { ... }
+Style/MultilineBlockChain:
+  Exclude:
+    - spec/**/*
+
+# Allow use of: something rescue nil
```

Comment:
Personally, I think `something rescue nil` is useful everywhere. But if you don't like `something rescue nil` to be used outside of specs, then sure, it can be changed.